### PR TITLE
CADC-9471: integrate feedback from April 7 demo

### PR DESCRIPTION
**Pull request by hjeeves**
includes:
- 'are you sure' delete confirmation
- tooltips for action affordances (delete, add, reload)
- last item deleted from session list never cleared from list
- remove 'connecting' modal (target tab gives enough information if the user can't connect to the requested session)

Added README.md and CHANGELOG.md here as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,62 @@
...
+ (currently notebook only,) allowing the amount of memory and number of cores you designate to be used to power
+ your session. 
 
 ## Endpoint locations
 All endpoints require authentication with CANFAR, and authorization to access Skaha resource allocations.
 
-Science Portal Session Launch
-http://www.canfar.net/science-portal/launch
+Science Portal URL
+http://www.canfar.net/science-portal
```
Comment (at88mph) [inline README.md]: Always use `https://`.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,62 @@
...
 More information can be found in the swagger docs: https://ws-uv.canfar.net/skaha
 
-## CANFAR Science Portal Workflow
+## CANFAR Science Portal Workflows
+All workflows assume you are logged in with a CADC account
+
+
+### Connecting to existing sessions
+
+1) From the main page: http://www.canfar.net/science-portal
```
Comment (at88mph) [inline README.md]: Ditto `https://`.

File: README.md [hunk 3]
```diff
@@ -1,38 +1,62 @@
...
+2) Science Portal will poll for and display any sessions you currently have
+3) Clicking on a session icon will connect to and forward you to the session
+
+
+### Launch a new session
+
+1) From the main page: http://www.canfar.net/science-portal
```
Comment (at88mph) [inline README.md]: Ditto `https://`.

File: README.md [hunk 4]
```diff
@@ -1,38 +1,62 @@
...
+(ie memory or # of cores)
+8) Select 'Launch'
+9) Science Portal will request the session be started, will add the session 
+to the list at the top of the page. 
+
+
+### Delete an existing session
 
-### Launching and Connecting to Notebook sessions
+1) From the main page: http://www.canfar.net/science-portal
```
Comment (at88mph) [inline README.md]: Ditto `https://`.

**Review by at88mph — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hjeeves) [reply README.md]: Thank you... end of day error there!

**Review by at88mph — APPROVED** [review]

**Pull request merged**